Bug#334516: Program termination message suggests the obsolete O command
rather than o to set options.

diff -Naur --exclude=debian perl-5.8.7.orig/lib/perl5db.pl perl-5.8.7/lib/perl5db.pl
--- perl-5.8.7.orig/lib/perl5db.pl	2004-12-30 10:21:10.000000000 +1100
+++ perl-5.8.7/lib/perl5db.pl	2005-10-29 07:12:16.000000000 +1000
@@ -2034,8 +2034,8 @@
             $term || &setterm;
             print_help(<<EOP);
 Debugged program terminated.  Use B<q> to quit or B<R> to restart,
-  use B<O> I<inhibit_exit> to avoid stopping after program termination,
-  B<h q>, B<h R> or B<h O> to get additional info.  
+  use B<o> I<inhibit_exit> to avoid stopping after program termination,
+  B<h q>, B<h R> or B<h o> to get additional info.  
 EOP
 
             # Set the DB::eval context appropriately.
